v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -119,7 +119,31 @@ def create_table(self) -> hl.MatrixTable:
             self.dataset_type,
             self.filters_path,
         )
-        mt = select_relevant_fields(mt, self.dataset_type)
+        mt = select_relevant_fields(
+            mt,
+            self.dataset_type,
+            [
+                *(
+                    # info.AF is "required" if the relatedness check
+                    # is expected to run.
+                    ['info.AF']
+                    if (
+                        self.check_sex_and_relatedness
+                        and self.dataset_type.check_sex_and_relatedness
+                    )
+                    else []
+                ),
+                *(
+                    # this field is never required, the pipeline
+                    # will run smoothly even in its absence, but
+                    # will trigger special handling if it is present.
+                    ['info.CALIBRATION_SENSITIVITY']
+                    if hasattr(mt, 'info')
```

Comment:
Do we need this line? If we just check `hasattr(mt.info, 'CALIBRATION_SENSITIVITY')` and `info` doesn't exist on `mt`, won't we will get a readable `AttributeError`?

I guess the difference is that if `info` doesn't exist on `mt` and we keep this line `hasattr(mt, 'info')`, the error would occur inside `select_relevant_fields`, which might be preferable.

Reply:
ah yeah, I think we don't want to throw an error if the field isn't present.  it's basically a flag for whether or not the callset is DRAGEN and is not expected in non-DRAGEN callsets.